src/plugins/discover/public/application/main/discover_main_route.tsx

```diff
@@ -54,15 +54,29 @@ export function DiscoverMainRoute({ services, history }: DiscoverMainProps) {
   const [indexPatternList, setIndexPatternList] = useState<
     Array<SavedObject<IndexPatternAttributes>>
   >([]);
-
+  const [isNewKibanaInstance, setNewKibanaInstance] = useState(false);
   const { id } = useParams<DiscoverLandingParams>();
 
+  useEffect(() => {
+    const fetchIsNewKibanaInstance = async () => {
+      const hasUserIndexPattern = await data.indexPatterns.hasUserDataView().catch(() => true);
```

Comment:
this this should be `data.dataViews`, isn't there a fee for every use of ******* (censored)?